### DISCUSSION TITLE
Shortcut to reply without subscribing to thread

### DISCUSSION THREAD

**Discussion by Johnnycyan** [Chatterino/chatterino2#4871] [Ideas]
Since there is the checkbox to unsubscribe from a thread it would be nice if when you send a reply you could just do like Alt+Enter or something else configurable to automatically send the message and unsubscribe from the thread.

There are many times when someone comes into the chat and I wanna say hi but I don't wanna get pinged for everyone else also saying hi in the thread.

**Comment by pajlada** [comment]
You could uncheck the "Automatically subscribe to participated reply threads", that way you'd have to opt in to subscribe to the threads rather than opt out

**Comment by Johnnycyan** [comment]
Well most of the time I want to be subscribed it's just in events like this where I want it off

**Reply by pajlada** [reply]
Not really feasible as a "send message & don't subscribe" hotkey since the concept of being subscribed comes after the message has been bounced back from Twitch. Best we could realistically do is a hotkey to toggle the "Automatically subscribe to participated reply threads" setting so you'd maybe do:

1. Press "Reply to message"
2. Type your message out
3. If you want to not subscribe to this thread, press CTRL+F11 or w/e
4. Send the message
5. Press CTRL+F11 to continue autosubscribe to threads

**Reply by Johnnycyan** [reply]
Ah I see. That could work then yeah.

**Reply by Johnnycyan** [reply]
By the way just wanted to say I was impressed with how quickly you replied. I appreciate you <3

**Comment by Johnnycyan** [comment]
And honestly it wouldn't be so bad but you can't unsubscribe from the thread if you're the first person to start the thread so I have to send the reply, then reopen the thread, then uncheck, and then close, every time.
